src/index.html

```diff
@@ -7,7 +7,53 @@
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Stars</h1>
+  <body class="container">
```

Comment:
Don't add classes to elements like `body`, it is unique and there is no reason to mark it additionally with classes, either style it using tag or use a container like `div`